app-games/migrant-demo/spec

```diff
@@ -0,0 +1,4 @@
+VER=0
```

Comment:
You may want to use 2249001 as the version number (assuming this is the build number).